internal/criocli/criocli.go

```diff
@@ -1092,7 +1092,7 @@ func getCrioFlags(defConf *libconfig.Config) []cli.Flag {
 		&cli.StringFlag{
 			Name:      "version-file-persist",
 			Usage:     "Location for CRI-O to lay down the persistent version file. It is used to check if crio wipe should wipe images, which should only happen when CRI-O has been upgraded.",
-			Value:     defConf.VersionFile,
+			Value:     defConf.VersionFilePersist,
```

Comment:
yeah @sohankunkerkar is correct, though you correctly found an issue here.

Reply:
the rest should be "" though

Reply:
maybe we should set the --version-file-persist to "", but not the "/var/run/crio/version" which is shown to user